include/dev/strainGauge.hpp

```diff
@@ -0,0 +1,45 @@
+#ifndef strainGauge_STRAINGAUGE_HPP
```

Comment:
Look at EVT-core for proper header guard naming conventions
```suggestion
#ifndef strainGauge_STRAINGAUGE_HPP
```